<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TaskManager</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>-4</x>
      <y>0</y>
      <width>591</width>
      <height>491</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>591</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>591</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">
</string>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_sys">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(10, 63, 91);
color: rgb(255, 255, 255);</string>
     </property>
     <attribute name="title">
      <string>System</string>
     </attribute>
     <widget class="QLabel" name="label_CPU">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>259</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Monaco</family>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>CPU Information</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_OS">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>391</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Monaco</family>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>OS Information</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>21</x>
        <y>41</y>
        <width>132</width>
        <height>111</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_1">
         <property name="font">
          <font>
           <family>Monaco</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="text">
          <string>CPU Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Monaco</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="text">
          <string>CPU Type: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Monaco</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="text">
          <string>CPU Freqency:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>41</y>
        <width>401</width>
        <height>111</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_CPU_name">
         <property name="font">
          <font>
           <family>Monaco</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="text">
          <string>Unknown</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_CPU_type">
         <property name="font">
          <font>
           <family>Monaco</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="text">
          <string>Unknown</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_CPU_freq">
         <property name="font">
          <font>
           <family>Monaco</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="text">
          <string>Unknown</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>281</y>
        <width>401</width>
        <height>111</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_OS_type">
         <property name="font">
          <font>
           <family>Monaco</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="text">
          <string>Unknown</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_OS_version">
         <property name="font">
          <font>
           <family>Monaco</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="text">
          <string>Unknown</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_GCC_version">
         <property name="font">
          <font>
           <family>Monaco</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>Unknown</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>22</x>
        <y>282</y>
        <width>131</width>
        <height>111</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Monaco</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="text">
          <string>OS Type:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>Monaco</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="text">
          <string>OS Version:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <family>Monaco</family>
           <pointsize>12</pointsize>
           <weight>50</weight>
           <bold>false</bold>
           <stylestrategy>PreferAntialias</stylestrategy>
          </font>
         </property>
         <property name="text">
          <string>GCC Version:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_mem">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(8, 88, 95);
color: rgb(255, 255, 255);</string>
     </property>
     <attribute name="title">
      <string>Memory</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_proc">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(8, 88, 95);
color: rgb(255, 255, 255);</string>
     </property>
     <attribute name="title">
      <string>Process</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_mod">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(8, 88, 95);
color: rgb(255, 255, 255);</string>
     </property>
     <attribute name="title">
      <string>Module</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_about">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(8, 88, 95);
color: rgb(255, 255, 255);</string>
     </property>
     <attribute name="title">
      <string>About</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(75, 9, 63);
color: rgb(255, 255, 255);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
